src/remoted/request.c

```diff
@@ -17,6 +17,7 @@
 #include "wazuh_modules/wmodules.h"
 
 #define COUNTER_LENGTH 64
+#define HC_FORCE_RECONNECT "force_reconnect"
```

Comment:
This is already defined in https://github.com/wazuh/wazuh/blob/fix/14797-get-force_reconnect-response/src/headers/rc.h#L45.